File: hospital-frontend/src/services/reviewService.js
import api from './api';

export const addReview = (reviewData) => {
    return api.post('/reviews', reviewData);
};

export const getReviewsByDoctor = (doctorId) => {
    return api.get(`/reviews/doctor/${doctorId}`);
};

export const deleteReview = (id) => {
    return api.delete(`/reviews/${id}`);
};

File: hospital-backend/src/main/java/com/hospital/hospital_backend/service/ReviewService.java
package com.hospital.hospital_backend.service;

import com.hospital.hospital_backend.dto.request.ReviewRequest;
import com.hospital.hospital_backend.entity.Review;
import java.util.List;

public interface ReviewService {
    Review addReview(ReviewRequest request);
    List<Review> getReviewsByDoctor(Long doctorId);
    void deleteReview(Long id);
}
